Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.yml

```diff
@@ -0,0 +1,2250 @@
+      type: Date
+    - contextPath: CarbonBlackDefense.Alert.last_event_time
+      description: The time of the latest event associated with the alert.
+      type: Date
+    - contextPath: CarbonBlackDefense.Alert.threat_id
+      description: The identifier of a threat that this alert belongs. Threats are
+        comprised of a combination of factors that can be repeated across devices.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.severity
+      description: The threat ranking of the alert.
+      type: Number
+    - contextPath: CarbonBlackDefense.Alert.category
+      description: The category of the alert. (THREAT, MONITORED).
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.device_id
+      description: The identifier assigned by Carbon Black Cloud to the device associated
+        with the alert.
+      type: Number
+    - contextPath: CarbonBlackDefense.Alert.device_os
+      description: The operating system of the device associated with the alert.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.device_os_version
+      description: The operating system and version on the device.
+      type: Unknown
+    - contextPath: CarbonBlackDefense.Alert.device_name
+      description: The hostname of the device associated with the alert.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.device_username
+      description: The username of the user logged on during the alert. If the user
+        is not available then this may be populated with the device owner.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.policy_id
+      description: The identifier for the policy associated with the device at the
+        time of the alert.
+      type: Number
+    - contextPath: CarbonBlackDefense.Alert.policy_name
+      description: The name of the policy associated with the device at the time of
+        the alert.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.target_value
+      description: The priority of the device assigned by the policy.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.workflow.state
+      description: The state of the tracking system for alerts as they are triaged
+        and resolved. Supported states are OPEN or DISMISSED.
+      type: String
+    - contextPath: CarbonBlackDefense.Alert.workflow.remediation
+      description: The state of the workflow of the tracking system for alerts as
+        they are triaged and resolved. Supported states are OPEN or DISMISSED.
+      type: Unknown
+    - contextPath: CarbonBlackDefense.Alert.workflow.last_update_time
+      description: The last time the alert was updated.
+      type: Date
+    - contextPath: CarbonBlackDefense.Alert.workflow.comment
+      description: The comment about the workflow of the tracking system for alerts
+        as they are triaged and resolved.
+      type: Unknown
```

Comment:
Also here.
Type 'unknown' should appear only when the output is a list or an object.